rustfmt.toml

```diff
@@ -0,0 +1,23 @@
+comment_width=120
+condense_wildcard_suffixes=true
+control_brace_style="ClosingNextLine"
+error_on_line_overflow=true
+error_on_unformatted=true
+fn_args_layout="Compressed"
+format_code_in_doc_comments=true
+format_macro_matchers=true
+format_strings=true
+hard_tabs=true
+match_arm_blocks=false
+match_block_trailing_comma=true
+max_width=120
+imports_granularity="Module"
```

Comment:
please keep only this line and remove the rest. we'll iterate over this as we see fit